web/blog/tenzir-v4.17/index.md

```diff
@@ -0,0 +1,125 @@
+---
+title: Tenzir v4.17
+authors: [dominiklohmann]
+date: 2024-06-21
+tags: [release, azure-log-analytics, lookup-table]
+comments: true
+---
+
+The new [Tenzir v4.17][github-release] brings an integration with Azure Log
+Analytics and introduces two mechanisms for expiring entries in lookup tables.
+
+![Tenzir v4.17](tenzir-v4.17.excalidraw.svg)
+
+[github-release]: https://github.com/tenzir/tenzir/releases/tag/v4.17.0
+
+<!-- truncate -->
+
+## Send Events to Azure Log Analytics
+
+The shining star of Tenzir v4.17 is the new [`azure-log-analytics` sink
+operator][azure-log-analytics-operator], which sends events to [Log Analytics in
+Azure Monitor][log-analytics-overview].
+
+:::tip Want to Learn More?
+We wrote an [integration guide][azure-log-analytics-integration] showing how to
+send your events zu Azure Log Analytics using Tenzir. Come check it out!
+:::
+
+[azure-log-analytics-operator]: /next/operators/azure-log-analytics
+[log-analytics-overview]: https://learn.microsoft.com/en-us/azure/azure-monitor/logs/log-analytics-overview
+[azure-log-analytics-integration]: /next/integrations/azure-log-analytics
+
+## Lookup Table Timeouts
+
+The `context update` operator gained two new options when used together with
+[lookup table contexts][lookup-table-docs]: `--create-timeout <duration>` and
+`--update-timeout <duration>`.
+
+Both new options cause individual events to expire in the lookup table. Create
+timeouts specify the time after which entries in the lookup table expire, and
+update timeouts specify the time after which entries in the lookup table expire
+when they're not accessed.
+
+```text {0} title="Expire lookup table entries after a week, or when not accessed in a day"
+…
+| context update my-lookup-table --create-timeout 1w --update-timeout 1d
```

Comment:
I think we need to be very clear that the minimum timeout is winning here. Saying "or" is not very clear.

Reply:
Do you think a "…, whichever expires first" helps?